invenio_pages/ext.py

```diff
@@ -53,8 +48,6 @@ def jinja_env(self):
         """Create a sandboxed Jinja environment."""
         if self._jinja_env is None:
             self._jinja_env = SandboxedEnvironment(
-                extensions=[
```

Comment:
is it ok to remove this?

Reply:
Yes, it's actually needed otherwise it will fail same for this [line](https://github.com/inveniosoftware/invenio-pages/pull/55/files/23c626b0ef1b8271e1c53712cc669b7961466197#diff-d8339c3a723fd8844f262fa97ca9fdb8ec6621d412cf34b58345cee76d17e3f1L34) 
See [changelog](https://jinja.palletsprojects.com/en/3.1.x/changes/#version-3-0-0)
> Deprecate the autoescape and with extensions, they are built-in to the compiler.